Automattic/wp-calypso
# Reader: Localize logged out tag and discover pages

project thread pe7F0s-148-p2

Applies i18n to logged out /tag and /discover pages.

### Existing issues

Note that on both pages, the first 7 posts are requested *before* the language is set, and therefore are in english. This is due to the language being set asynchronously, I will fix this in antother PR https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/i18n-utils/switch-locale.js#L435

### Testing instructions 
Sansbox public-api and apply D118322-code for the tag pages
With a logged out session:
try `/fr/discover` discover page should be localized.
try `/fr/tag/art` art page should be localized.
invalid languages should redirect to the english page e.g. `/zz/tag/art` -> `/tag/art`
The language selector in the footer should switch the language.
